test/rlx_extended_bin_SUITE.erl

```diff
@@ -579,6 +581,120 @@ replace_os_vars(Config) ->
                           {"COOKIE", "cookie2"}]),
     ok.
 
+
+replace_os_vars_sys_config_vm_args_src(Config) ->
+    LibDir1 = proplists:get_value(lib1, Config),
+
+    rlx_test_utils:create_app(LibDir1, "goal_app", "0.0.1", [stdlib,kernel], []),
+
+    ConfigFile = filename:join([LibDir1, "relx.config"]),
+    SysConfigSrc = filename:join([LibDir1, "sys.config.src"]),
+    VmArgs = filename:join([LibDir1, "vm.args.src"]),
+
+    rlx_test_utils:write_config(ConfigFile,
+                 [{release, {foo, "0.0.1"},
+                   [goal_app]},
+                  {lib_dirs, [filename:join(LibDir1, "*")]},
+                  {sys_config_src, SysConfigSrc},
+                  {vm_args_src, VmArgs},
```

Comment:
would it be worth having the test (or a version of the test) specify both <thing>_src and <thing> to check that priorities are taking place properly? I think this test is fine to cover the base case, but the corner case is possibly not covered right now, even though it's obvious through the code review that it should work.

Reply:
Yea, good point. Will add.